### PR TITLE
Feature/initial app setup

### PR DESCRIPTION
**Pull request by priyaravi23**

### CODE REVIEW

File: client/src/utils/queries.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import gql from 'graphql-tag';
+
+export const QUERY_COMMENTS = gql`
```
Comment (vanessalane) [inline client/src/utils/queries.js:3]: is this used anywhere?

File: server/schemas/typeDefs.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        image: String
+        release: String
+    }
+    type Comment {
+        _id: ID
+        commentText: String
+        createdAt: String
+        username: String
+        reactionCount: Int
+        reactions: [Reaction]
```
Comment (vanessalane) [inline server/schemas/typeDefs.js:37]: should Comment include the movie Id?

File: client/src/components/HomePage.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+        return (
+            <section className="my-5 homepage">
+                <div className="home-container">
+
+                    <p className="display-4 home-text">Track your favorite movies</p>
+
+                    <Form onSubmit={(event) => {this.props.handleFormSubmit(event, this.state.searchInput)}}>
+                        <div className="home-input">
+                            <InputGroup className="mb-3">
```
Comment (vanessalane) [inline client/src/components/HomePage.js:34]: You might be able to use Form.Group here instead of having to import InputGroup

**Review by anitapeppercorn — APPROVED** [review]

**Comment by anitapeppercorn** [comment]
Start of

**Pull request merged**